internal/cmd/marketplace/get.go

```diff
@@ -94,19 +113,26 @@ func getMarketplaceItemByID(client *client.APIClient, resourceID string) (*marke
 	return marketplaceItem, nil
 }
 
-// getMarketplaceResource retrieves the marketplace items for a given resource ID
-func getMarketplaceResource(client *client.APIClient, resourceID string, outputFormat string) error {
-	marketplaceItem, err := getMarketplaceItemByID(client, resourceID)
+// getItemEncodedWithFormat retrieves the marketplace item corresponding to the specified identifier, serialized with the specified outputFormat
+func getItemEncodedWithFormat(client *client.APIClient, objectID, companyID, itemID, version, outputFormat string) (string, error) {
+	var item *marketplace.Item
+	var err error
+	if objectID != "" {
+		item, err = getItemByObjectID(client, objectID)
+	} else {
+		if companyID == "" {
+			return "", marketplace.ErrMissingCompanyID
+		}
+		item, err = getItemByItemIDAndVersion(client, companyID, itemID, version)
```

Comment:
What is supposed to happen if itemId or version is missing?

Reply:
It's impossible they are missing at this point, the command flags are marked as required